src/Elliptic/Systems/Elasticity/Tags.hpp

```diff
@@ -0,0 +1,60 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+/*!
+ * \ingroup EllipticSystemsGroup
+ * \brief Items related to solving linear elasticity problems
+ *
+ * \details In linear elasticity problems we solve for the displacement vector
+ * field \f$\boldsymbol{u}\f$ in an elastic material that responds to an applied
+ * force \f$\boldsymbol{F}\f$. In this static approximation the equation of
+ * motion reduces to the elliptic equation \f[
+ * -\nabla_i T^{ij} = F^j
+ * \f]
+ * where the stress \f$T^{ij}\f$ is related to the strain
+ * \f$S_{ij}=\nabla_{(i}u_{j)}\f$ by a linear constitutive relation
+ * \f$T^{ij}=\chi^{ijkl}S_{kl}\f$ that describes the elastic properties of the
+ * material.
+ */
+namespace Elasticity {
+
+/*!
+ * \brief The amount of material deformation \f$\boldsymbol{u}(x)\f$ when a
+ * static force is applied.
+ */
+template <size_t Dim>
+struct Displacement : db::SimpleTag {
+  using type = tnsr::I<DataVector, Dim>;
```

Comment:
Most thermal-noise applications I know of use an orthonormal basis, so it's simplest to just keep all the indices on everything down. But maybe this doesn't fit well with how this will generalize to XCTS?

Reply:
Indeed I think the current setup up tensor types makes most sense for XCTS (and also elasticity). Not 100% sure, but my current understanding.